### PR TITLE
feat：流水线插件名称和简介限制优化 #8319

### PR DESCRIPTION
**Pull request by yjieliang**

### CODE REVIEW

File: src/backend/ci/core/common/common-web/src/main/kotlin/com/tencent/devops/common/web/constant/BkStyleEnum.kt [hunk 1]
```diff
@@ -34,6 +34,8 @@ enum class BkStyleEnum(val style: String) {
     ID_STYLE("^[\\w-]{1,64}\$"), // 标识正则表达式
     NAME_STYLE("^[\\w-\\u4E00-\\u9FBB\\u3400-\\u4DBF\\uF900-\\uFAD9\\u3000-\\u303F" +
         "\\u2000-\\u206F\\uFF00-\\uFFEF.()]{1,30}\$"), // 名称正则表达式
+    ATOM_NAME_STYLE("^[\\w-\\u4E00-\\u9FBB\\u3400-\\u4DBF\\uF900-\\uFAD9\\u3000-\\u303F" +
```
Comment (carlyin0801) [inline src/backend/ci/core/common/common-web/src/main/kotlin/com/tencent/devops/common/web/constant/BkStyleEnum.kt:37]: 不要加一个专门的名称正则表达式，其它组件的名称也要支持英文，基于原来的NAME_STYLE改吧，现在是条件放宽，不会对原来的逻辑产生影响

**Comment by tencentblueking-adm** [comment]
[CLA assistant check](https://bk-cla.bktencent.com:443/TencentBlueKing/bk-ci?pullRequest=8344) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://bk-cla.bktencent.com:443/TencentBlueKing/bk-ci?pullRequest=8344) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://bk-cla.bktencent.com:443/check/TencentBlueKing/bk-ci?pullRequest=8344) it.</sub>

File: src/backend/ci/core/store/api-store/src/main/kotlin/com/tencent/devops/store/pojo/atom/MarketAtomUpdateRequest.kt [hunk 2]
```diff
@@ -39,6 +41,7 @@ data class MarketAtomUpdateRequest(
     @ApiModelProperty("插件代码", required = true)
     val atomCode: String,
     @ApiModelProperty("插件名称", required = true)
+    @field:BkField(patternStyle = BkStyleEnum.NAME_STYLE)
```
Comment (carlyin0801) [inline src/backend/ci/core/store/api-store/src/main/kotlin/com/tencent/devops/store/pojo/atom/MarketAtomUpdateRequest.kt:44]: 更新接口先不要加这个校验，以前一些老插件的名称并不符合这个要求，新增插件接口可以加

**Review by carlyin0801 — APPROVED** [review]
ok

**Review by irwinsun — APPROVED** [review]

**Pull request merged**